src/schema/v2/__tests__/partner.test.js

```diff
@@ -689,4 +689,71 @@ describe("Partner type", () => {
       })
     })
   })
+
+  describe("#articlesConnection", () => {
+    let articlesResponse
+
+    beforeEach(() => {
+      articlesResponse = {
+        count: 4,
```

Comment:
#minor

This test would be a little more "realistic" if the count matched the number of results here, so I'd recommend adding one more result to the list. That would also ensure that our `first: 3` is working as expected.